ext/storage_sources/s3_store/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 project(s3_store CXX)    
 
 # Create an intermediate library to build the helper library.
-add_library(aws_bucket_util STATIC aws_bucket_conn.cxx)
+add_library(aws_bucket_util STATIC S3Connection.cpp)
```

Comment:
Please don't rename the files in the camel case, the new name could be ```s3_connection.cpp``` . Similarly ```s3_storage_source.cpp```